package/opt/hassbian/suites/files/webterminalsslhelper.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# Helper script for using LE certificates with Webterminal (shellinabox)
+if [ -d "/etc/letsencrypt/live" ]; then
+  CERTDIR="/etc/letsencrypt/live/"
+elif [ -d "/home/homeassistant/dehydrated/certs" ]; then
+  CERTDIR="/home/homeassistant/dehydrated/certs/"
+else
+  CERTDIR=""
+fi
+FULLCHAIN=$(find "$CERTDIR" -type f | grep fullchain)
+PRIVKEY=$(find "$CERTDIR" -type f | grep privkey)
+DOMAIN=$(ls "$CERTDIR")
+cat "$FULLCHAIN" "$PRIVKEY" > /var/lib/shellinabox/certificate-"$DOMAIN".pem
```

Comment:
Doing it this way would not link the cert but only "copy" it. Preferable would be to create a symlink to the certificate so that it's linked to the used certificate.

Reply:
In every guide I have found the cert file used for shellinabox has to include both certificate and privkey, is it possible to merge with linking?

Reply:
I missed that. That would require some strange magic.
Most likely this will stop working as soon as the certificate expires. If your ok with that then we can merge this.
Could you add to the update function to update the certificate if it's there?

Reply:
I hope not :( that defeats the purpose of the separate webterminalhelper.sh script..
Should't [this] update the cert for shellinabox 1AM every night?

"Could you add to the update function to update the certificate if it's there?"
Do you mean the update for dehydrated with the DuckDNS script, then no. that will result in certbot certs not being updated.
If this was not what you ment, enlighten me :D

Reply:
I missed that part of the webterminalhepler.sh script.

This look great now!